common/cond_handle.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#ifndef COMMON_COND_HANDLE_H_
+#define COMMON_COND_HANDLE_H_
+
+/**
+ * We declare a type cond_handle_t that depends on the system conditional
```

Comment:
The use of "condition variable" should be more common than "conditional variable".
Reference: https://en.wikipedia.org/wiki/Monitor_(synchronization)

Reply:
You're right, I don't even know why I called them like that in the first place

Reply:
Done, thanks